src/mol-io/reader/cif/schema.ts

```diff
@@ -53,11 +53,18 @@ function getColumnCtor(t: Column.Schema): ColumnCtor {
     }
 }
 
+function hasPresentValues(rowCount: number, valueKind: (row: number) => Column.ValueKind) {
+    for (let i = 0, il = rowCount; i < il; i++) {
+        if (valueKind(i) === Column.ValueKind.Present) return true;
+    }
+    return false;
+}
+
 function createColumn<T>(schema: Column.Schema, field: Data.CifField, value: (row: number) => T, toArray: Column<T>['toArray']): Column<T> {
     return {
         schema,
         __array: field.__array,
-        isDefined: field.isDefined,
+        isDefined: field.isDefined && hasPresentValues(field.rowCount, field.valueKind),
```

Comment:
I would prefer for this to be just part of the ``normalizeAtomSite`` function, as ``isDefined``'s sematics is essentially if it was present in the input file, not whether it has all values "undefined/unknown".